docker/README.md

```diff
@@ -82,10 +82,27 @@ You’ll also need to set the environment variable `OP_LETSENCRYPT_DOMAIN` withi
 
 Ensure that `OP_LETSENCRYPT_DOMAIN` is set to the domain name you’ve set up before continuing.
 
+(Optional) If you’re part of the Google Workspace provisioning beta, you will need to set up some additional secrets to use this functionality. Refer to our complete Google Workspace provisioning beta documentation for more complete information.
+
+First, you will want to edit the file located at `scim-examples/beta/workspace-settings.json` and enter in the appropriate details.
```

Comment:
For clarity and readability, I would suggest removing "you will want to" and other similar phrases throughout.